### ISSUE TITLE
Integrate a flaky spec tracker to automate detection of flaky tests

### ISSUE THREAD

**Issue by mirie** [forem/forem#18737]
Candidate to try out: https://buildpulse.io/

<img width="1077" alt="Screen Shot 2022-11-16 at 12 40 12 PM" src="https://user-images.githubusercontent.com/896728/202253674-16ee1a54-9d43-4a78-8404-99216bd613e3.png">

We'll likely want to use the startup level as the first step and also set a reminder to review before trial period ends to determine if we want to continue with the service (Trial is 14 days)

**Comment by mirie** [comment]
@maestromac did you already get payment stuff sorted? Was about to ask @jessleenyc in Slack, but I see you've already done some integration work so are you unblocked?
